src/routes/LocationPage/index.js

```diff
@@ -1,34 +1,38 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import Hierarchy from './Hierarchy'
+
 import { connect } from 'react-redux'
+import compose from 'lodash/fp/compose'
 
 import Content from 'components/Content'
 import Breadcrumb from 'components/Content/Breadcrumb'
 import RichLayout from 'components/RichLayout'
 import Error from 'components/Error'
-import { PageFetcher } from 'endpoints'
 import PageModel from 'endpoints/models/PageModel'
-import PDFButton from '../../components/Content/PDFButton'
+import PDFButton from 'components/Content/PDFButton'
+import withFetcher from 'endpoints/withFetcher'
+import PAGE_ENDPOINT from 'endpoints/page'
+
+import Hierarchy from './Hierarchy'
 
-class PageAdapter extends React.Component {
+class ContentWrapper extends React.Component {
   static propTypes = {
     location: PropTypes.string.isRequired,
     language: PropTypes.string.isRequired,
     path: PropTypes.string,
```

Comment:
why not required 🤔 ?

Reply:
because it's not always available. For example in route: /location/augsburg/de